### ISSUE TITLE
Does this code use the gpu ?

### ISSUE THREAD

**Issue by hyperfraise** [aleju/imgaug#178]
Hi. I just found out that opencv-python doesn't actually use the gpu, even when properly compiled with cuda support on.

I'm currently actively looking for a package that does so. Does this repo expose a code that makes augmentations run on gpu ?

**Comment by aleju** [comment]
No, the library currently only works on cpu. There is an augmentation library from nvidia which uses the gpu, but I forgot the name.

**Comment by alar0330** [comment]
> 
> 
> No, the library currently only works on cpu. There is an augmentation library from nvidia which uses the gpu, but I forgot the name.

NVIDIA DALI is the augmentation library and the accompanying JPEG-decoder on GPU is NVIDIA nvJPEG.
